stanford-mods.gemspec

```diff
@@ -12,29 +12,20 @@ Gem::Specification.new do |gem|
 
   gem.extra_rdoc_files = ["LICENSE", "README.md"]
   gem.files         = `git ls-files`.split($\)
-  gem.executables   = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
+  gem.executables   = gem.files.grep(%r{^bin/}).map { |f| File.basename(f) }
   gem.test_files    = gem.files.grep(%r{^spec/})
   gem.require_paths = ["lib"]
 
   gem.add_dependency 'mods', '~> 2.0.2'
   # active_support for .ordinalize, eg. 1 -> 1st, 2 -> 2nd for centuries
-  gem.add_dependency 'activesupport'
-
-  # Runtime dependencies
-  # gem.add_runtime_dependency 'nokogiri'
+  gem.add_dependency 'activesupport', '~> 4.2'
```

Comment:
I thought the Rails 5.x problem was because of the ruby support matrix?  (and I'm not seeing any failures with activesupport 5.x locally, which is why I ask..)